### PR TITLE
Homework_17

### PR DESCRIPTION
**Pull request by VaroAGHA**
Add song to playlist

### CODE REVIEW

**Comment by VaroAGHA** [comment]
Homework__17/ Add Song To Playlist

File: src/test/java/Homework_17.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework_17 {
```
Comment (Berichicko) [inline src/test/java/Homework_17.java]: **According to the Java style guide:** 

1. Class names should be nouns, in mixed cases with the first letter of each internal word capitalized. Interfaces names should also be capitalized just like class names.
2. Use whole words and must avoid acronyms and abbreviations.

For example public class **HomeWork17**

File: src/test/java/Homework_17.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+//public class Homework17 extends BaseTest {
+
+    @Test
+    public void addSongToPlaylist() {
```
Comment (Berichicko) [inline src/test/java/Homework_17.java]: If possible, avoid overly long methods. A long method is 20‒25 lines. If possible, you should split the method into several shorter ones or even create a new class for this.

**Comment by Berichicko** [comment]
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

image

HINT

**The code you posted contains Git conflict markers (<<<<<<< Updated upstream and >>>>>>> Stashed changes). These markers indicate that there are conflicting changes in the code that need to be resolved.**

Resolve Git conflicts https://www.jetbrains.com/help/idea/resolve-conflicts.html

or

Resolving a merge conflict on GitHub https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github

or

How to resolve a Merge Conflict in GitHub Desktop https://www.youtube.com/watch?v=MIVW0sijSjY

**Comment by VaroAGHA** [comment]
Hello Aliaksandr

I have a problem with Resolve Git conflicts. We didn't learn this point during the classes, please help me to finish Homework17.
________________________________

**Comment by Berichicko** [comment]
> Hello Aliaksandr I have a problem with Resolve Git conflicts. We didn't learn this point during the classes, please help me to finish Homework17.

Hello

Please, re-submit homework 17 for review and I will try to help you

**Comment by VaroAGHA** [comment]
Thank you! I did.
________________________________

**Pull request closed without merging**